build.gradle.kts

```diff
@@ -205,7 +205,7 @@ tasks {
     return destinationDir
   }
 
-  val codyCommit = "68281675fd0731c0b2d30f61b85bb84dea165242"
```

Comment:
https://github.com/sourcegraph/cody/commit/8a61ffc8a55631350a31276b39f38e003007a3d2

Reply:
https://github.com/sourcegraph/cody/commit/65cbc12c00aed56c4013ef6c97736a82ac0dc294

Reply:
Out of curiosity why there is an hardcoded commit hash istead of branch or tag name?